scripts/preBuildHTML.py

```diff
@@ -1,4 +1,17 @@
-import os
+from subprocess import call
+import os.path
+
 
 def preBuildHTMLFun():
-    os.system("npm run build")
\ No newline at end of file
+    webpackInst = os.path.isdir("../node_modules/webpack")

```

Comment:
I would check for the existence of the node_modules folder only, and not the webpack subfolder. You can also install webpack globally, and I am not sure if it will show up in the node_modules folder of an individual repository in that case

Reply:
I did some tests and even with webpack installed globally it will be installed locally when running `npm ci` command. 

![webpack global vs local](https://user-images.githubusercontent.com/2778501/125839181-bc4910c3-22ff-49c6-a707-02f7cb90b053.png)